[ERROR] 2018-01-31 21:40:47,287 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`owner`\",\"`tabSaisie suivi OF`.`creation`\",\"`tabSaisie suivi OF`.`modified`\",\"`tabSaisie suivi OF`.`modified_by`\",\"`tabSaisie suivi OF`.`_user_tags`\",\"`tabSaisie suivi OF`.`_comments`\",\"`tabSaisie suivi OF`.`_assign`\",\"`tabSaisie suivi OF`.`_liked_by`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`parent`\",\"`tabSaisie suivi OF`.`parenttype`\",\"`tabSaisie suivi OF`.`parentfield`\",\"`tabSaisie suivi OF`.`idx`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\",\"`tabSaisie suivi OF`.`sof_flagcloture`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
[ERROR] 2018-01-31 21:45:05,777 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
[ERROR] 2018-01-31 21:45:06,098 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
[ERROR] 2018-01-31 21:45:46,057 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
[ERROR] 2018-01-31 21:45:47,281 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`owner`\",\"`tabSaisie suivi OF`.`creation`\",\"`tabSaisie suivi OF`.`modified`\",\"`tabSaisie suivi OF`.`modified_by`\",\"`tabSaisie suivi OF`.`_user_tags`\",\"`tabSaisie suivi OF`.`_comments`\",\"`tabSaisie suivi OF`.`_assign`\",\"`tabSaisie suivi OF`.`_liked_by`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`parent`\",\"`tabSaisie suivi OF`.`parenttype`\",\"`tabSaisie suivi OF`.`parentfield`\",\"`tabSaisie suivi OF`.`idx`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\",\"`tabSaisie suivi OF`.`sof_flagcloture`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
[ERROR] 2018-01-31 21:45:47,592 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`owner`\",\"`tabSaisie suivi OF`.`creation`\",\"`tabSaisie suivi OF`.`modified`\",\"`tabSaisie suivi OF`.`modified_by`\",\"`tabSaisie suivi OF`.`_user_tags`\",\"`tabSaisie suivi OF`.`_comments`\",\"`tabSaisie suivi OF`.`_assign`\",\"`tabSaisie suivi OF`.`_liked_by`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`parent`\",\"`tabSaisie suivi OF`.`parenttype`\",\"`tabSaisie suivi OF`.`parentfield`\",\"`tabSaisie suivi OF`.`idx`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\",\"`tabSaisie suivi OF`.`sof_flagcloture`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
[ERROR] 2018-01-31 21:45:55,662 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`owner`\",\"`tabSaisie suivi OF`.`creation`\",\"`tabSaisie suivi OF`.`modified`\",\"`tabSaisie suivi OF`.`modified_by`\",\"`tabSaisie suivi OF`.`_user_tags`\",\"`tabSaisie suivi OF`.`_comments`\",\"`tabSaisie suivi OF`.`_assign`\",\"`tabSaisie suivi OF`.`_liked_by`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`parent`\",\"`tabSaisie suivi OF`.`parenttype`\",\"`tabSaisie suivi OF`.`parentfield`\",\"`tabSaisie suivi OF`.`idx`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\",\"`tabSaisie suivi OF`.`sof_flagcloture`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
[ERROR] 2018-01-31 21:45:56,673 | /home/frappe/frappe-bench/apps/frappe/frappe/app.py:
Site: Suivi_Atelier
Form Dict: {
 "cmd": "frappe.desk.reportview.get", 
 "doctype": "Saisie suivi OF", 
 "fields": "[\"`tabSaisie suivi OF`.`name`\",\"`tabSaisie suivi OF`.`owner`\",\"`tabSaisie suivi OF`.`creation`\",\"`tabSaisie suivi OF`.`modified`\",\"`tabSaisie suivi OF`.`modified_by`\",\"`tabSaisie suivi OF`.`_user_tags`\",\"`tabSaisie suivi OF`.`_comments`\",\"`tabSaisie suivi OF`.`_assign`\",\"`tabSaisie suivi OF`.`_liked_by`\",\"`tabSaisie suivi OF`.`docstatus`\",\"`tabSaisie suivi OF`.`parent`\",\"`tabSaisie suivi OF`.`parenttype`\",\"`tabSaisie suivi OF`.`parentfield`\",\"`tabSaisie suivi OF`.`idx`\",\"`tabSaisie suivi OF`.`sof_perscode`\",\"`tabSaisie suivi OF`.`sof_ofcode`\",\"`tabSaisie suivi OF`.`sof_codart`\",\"`tabSaisie suivi OF`.`sof_numoper`\",\"`tabSaisie suivi OF`.`sof_mainoeuvre`\",\"`tabSaisie suivi OF`.`sof_desi`\",\"`tabSaisie suivi OF`.`sof_qteproduite`\",\"`tabSaisie suivi OF`.`sof_ofcloture`\",\"`tabSaisie suivi OF`.`sof_flag`\",\"`tabSaisie suivi OF`.`sof_flaghisto`\",\"`tabSaisie suivi OF`.`sof_flagcloture`\"]", 
 "filters": "[]", 
 "order_by": "`tabSaisie suivi OF`.`modified` desc", 
 "page_length": "20", 
 "start": "0", 
 "with_comment_count": "true"
}
Request Error
Traceback (most recent call last):
  File "/home/frappe/frappe-bench/apps/frappe/frappe/app.py", line 62, in application
    response = frappe.handler.handle()
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 22, in handle
    data = execute_cmd(cmd)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/handler.py", line 53, in execute_cmd
    return frappe.call(method, **frappe.form_dict)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/__init__.py", line 939, in call
    return fn(*args, **newargs)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 21, in get
    data = compress(execute(**args), args = args)
  File "/home/frappe/frappe-bench/apps/frappe/frappe/desk/reportview.py", line 85, in compress
    new_row.append(row[key])
KeyError: '_comment_count'
